code/modules/mob/living/carbon/human/say.dm

```diff
@@ -145,11 +145,15 @@
 		if(silent || HAS_TRAIT(src, TRAIT_MUTE))
 			S.message = ""
 
+		if(wear_mask)
+			var/speech_verb_when_masked = wear_mask.change_speech_verb()
+			if(speech_verb_when_masked)
+				verb = speech_verb_when_masked
```

Comment:
It is best to place this part of the code before the for loop.
Seeing `verb` is never used in the for loop and it never changes after one iteration.
Doing this also retains the behaviour of being brain damaged changing the verb even if you have a horsemask on... Bug? Your say